docker/webui/haproxy-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+cp /usr/local/etc/haproxy/haproxy.cfg.template /usr/local/etc/haproxy/haproxy.cfg
+for i in $(seq 2 $replicas); do
+  echo "  server webui$i webui_webui_$i:9526" >>/usr/local/etc/haproxy/haproxy.cfg
+done
+
+/usr/local/sbin/haproxy -f /usr/local/etc/haproxy/haproxy.cfg
```

Comment:
You're launching a binary from `/usr/local`? So this is using a custom build of haproxy instead of the packaged version? I'm just wondering because at least under TW the path would just be `/usr/sbin/haproxy`.

Reply:
Well, the default docker image for package only have this in 

```
root@eab356a574de:/# whereis haproxy
haproxy: /usr/local/sbin/haproxy /usr/local/etc/haproxy
root@eab356a574de:/# /user/sbin/haproxy 
bash: /user/sbin/haproxy: No such file or directory
```

Reply:
Ok, so the Docker image you're using relies on a custom build? Mh… doesn't sound great but I guess it isn't very important (at this point).